console/src/components/UIKit/atoms/Icon/index.tsx

```diff
@@ -32,7 +32,11 @@ const iconReferenceMap = {
 const iconWidth = 18;
 const iconHeight = 18;
 
-export const Icon = props => {
+export type IconProps = {
+  type: keyof typeof theme.icon;
```

Comment:
As PR converting theme config to TypeScript was just merged, you may want to use `Theme['icon']`, where `Theme` is an interface exported from `theme.js`.

Reply:
I fixed it.
https://github.com/hasura/graphql-engine/pull/4373/commits/aa4605cbfd8f6ebc2f5c279edc131a8ceb51636d